apps/meteor/app/discussion/server/hooks/propagateDiscussionMetadata.ts

```diff
@@ -100,3 +100,13 @@ callbacks.add(
 	callbacks.priority.LOW,
 	'CleanDiscussionMessage',
 );
+
+callbacks.add(
+	'afterDeleteRoom',
+	async (rid) => {
+		await ReadReceipts.removeByRoomId(rid);
```

Comment:
this file is not the best fit for this action, since this is not related to discussions.